junit-vintage-engine/src/main/java/org/junit/vintage/engine/VintageTestEngine.java

```diff
@@ -70,17 +86,107 @@ public void execute(ExecutionRequest request) {
 		engineExecutionListener.executionStarted(engineDescriptor);
 		RunnerExecutor runnerExecutor = new RunnerExecutor(engineExecutionListener,
 			engineDescriptor.getTestSourceProvider());
-		executeAllChildren(runnerExecutor, engineDescriptor);
+
+		final ConfigurationParameters configurationParameters = request.getConfigurationParameters();
```

Comment:
We don't use `final` for local variables and method parameters. Please remove those.

Reply:
I made this change.